tools/jira-integration/main.go

```diff
@@ -0,0 +1,63 @@
+// Command jira-integration tests creating a ticket to a Jira instance via
+// the Fleet worker processor. It creates it exactly as if a Jira integration
+// was configured and a new CVE and related CPEs was found.
+//
+// Note that the Jira user's password must be provided via an environment
+// variable, JIRA_PASSWORD.
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/fleetdm/fleet/v4/server/service/externalsvc"
+	"github.com/fleetdm/fleet/v4/server/worker"
+	kitlog "github.com/go-kit/kit/log"
+)
+
+func main() {
+	var (
+		jiraURL        = flag.String("jira-url", "https://fleetdm.atlassian.net/", "The Jira instance URL")
```

Comment:
Let's remove the default to not expose fleet's JIRA test endpoint.

Reply:
Fixed in 43ab9074e